docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.8'
+services:
+    todo-app:
+        build:
+            context: .
+            dockerfile: Dockerfile
+            target: development
+        ports:
+            - '5000:5000'
+        env_file: .env
+        
```

Comment:
The exercise brief states that we want the development image to use hot reloading. So if you make a change to the files you should be able to see that change (nearly) immediately without having to rebuild the image or even restart the container.

> The solution is to use a bind mount when running the container to make the project directory on your host machine available as a mounted directory within the container.

You need to update this file and the Dockerfile to achieve that. The COPY command can then also be moved into the production stage specifically.